ortools/xpress/environment.cc

```diff
@@ -244,7 +236,9 @@ absl::Status LoadXpressDynamicLibrary(std::string& xpresspath) {
     }
 
     if (xpress_library.LibraryIsLoaded()) {
-      xpress_load_status = LoadXpressFunctions(&xpress_library);
+      LOG(INFO) << "Loading all Xpress functions";
+      LoadXpressFunctions(&xpress_library);
```

Comment:
Error handling isn't so great here. Shouldn't we return an `absl::status` here ?

Reply:
I cannot find a way to do it.
`LoadXpressFunctions` calls `DynamicLibrary.GetFunction(const char* function_name) `

`DynamicLibrary.GetFunction(const char* function_name) ` executes the `CHECK` macro that seems to throw a fatal error. I tried to catch it but i probably did not figure out how to do it properly.
Gurobi interface uses the same approach you see here and in the end we can consider that
not having Xpress installed is ok
having the wrong xpress installation is worse

Reply:
I open to change this behavior but at the moment I cannot see how to do it without touching the `dynamic_library.h` file